spec/tools/environment_builders/openstack/services/storage/data.rb

```diff
@@ -0,0 +1,33 @@
+require_relative '../base_data'
+
+module Openstack
+  module Services
+    module Storage
+      class Data < ::Openstack::Services::BaseData
+        def directory_name_1
+          "dir_1"
```

Comment:
maybe ivar this?  `@directory_name_1 ||= "dir_1"`?  It's only in spec tools, so I'm not terribly worried about it